source/renderer/app/components/staking/stake-pools/StakePoolsSearch.js

```diff
@@ -64,6 +64,7 @@ type Props = {
   onGridRewardsView?: Function,
   onListView?: Function,
   search: string,
+  isMainnet?: boolean,
```

Comment:
@aleksandardjordjeviciohk please make this required property

Reply:
updated